Release notes XTC 69 (#196)
Xceptance/xlt-documentation: content/en/xtc/xtc-release-notes/v69.md

---
title: XTC v69
linkTitle: XTC v69

weight: 147
type: docs

date: 2023-07-17

description: 2023-07-18

sitemap:
  changefreq: never
  priority: 0.1
---

## Platform

### Project States

The states a project may have in XTC have been refined and enhanced further. In this course, project states have evolved from a simple active/locked marker to a set of different states that influence not only what can be done with the project, but also whether the project is visible at all.

Projects may now have three different states:

> * **Active** – The project is visible to all project members. They can act within the project as allowed by their roles. This is the normal state while people work actively on the project.

Reviewer: As 'act' also has the meaning of 'behave', it might not be right verb here.
```suggestion
* **Active** – The project is visible to all project members. They can interact with the project as allowed by their roles. This is the normal state while people work actively on the project.
```

Author: Applied as suggested.

* **Inactive** – The project is visible to all project members, but the project is effectively read-only. No object can be created or modified. In monitoring projects, scenario execution is halted, while in load testing projects, no load tests can be started. Resources shared with other people stay shared. Set a project to inactive if the work is done but you still want members to be able to recap what has been done and review the results.

> * **Archived** – The project is effectively invisible. It will be displayed neither on your dashboard, nor in the project select drop-down at the top of a page. Shared resources will be unshared, but no data will be deleted. Set a project to archived if the project is complete and members won’t need to look into it any longer.

Reviewer: ```suggestion
* **Archived** – The project is effectively invisible. It will be displayed neither on your dashboard, nor in the page header's project drop-down. Share-Links to project resources will be removed from the system and thus will become invalid. No data will be deleted. Set a project to archived if the project has completed and members won’t need to look into it any longer.
```

Author: Applied with modifications.

The state of a project can be altered as follows:

* A project administrator may toggle the state of the project between active and inactive. In the project, go to *Configuration* > *General*  > *Project State* and edit the state as needed.

> * An organization administrator may set all three project states. In the organization, go to *Projects*, click the *Change State* menu item in the 3-dot menu of the project in question, and set the state as needed.

Reviewer: ```suggestion
* An organization administrator may set all three project states. In the organization, go to *Projects*, click the *Change State* menu item in the 3-dot menu of the project in question and set the state as needed.
```

Author: I would like to keep the text as is.

> You can change the state of a project at any time. For example, if a project needs more work, it can be set to active again, from state inactive, or even archived.

Reviewer: ```suggestion
You can change the state of a project at any time. For example, if a project needs more work, it can be set to active  again, no matter whether it's inactive or even archived.
```

Author: I would like to keep the text as is.

Author: Done.

Note that the new project states do not influence the base fee for a project. This fee will apply to any state.

## Load Testing

> * When creating a new load test report, you may now define a custom time zone. This affects all time values shown in the report, especially in the charts. In the *Create Load Test Report* dialog, click the *Show Advanced Settings* button and specify the desired time zone in the *Misc Settings* section, for example, *EST*. If no time zone is given, XTC will use UTC by default.

Reviewer: ```suggestion
* When creating custom load test reports, you now have the option to specify the time zone to be used.This affects all time values shown in the report, especially in the charts. In the *Create Load Test Report* dialog, click the *Show Advanced Settings* button and specify the desired time zone in the *Misc Settings* section, for example, *EST*. XTC will use UTC if no time zone is given.
```

Author: Applied in parts.
